city_scrapers/spiders/chi_ssa_64.py

```diff
@@ -0,0 +1,89 @@
+import html
+import re
+from datetime import datetime
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa64Spider(CityScrapersSpider):
+    name = "chi_ssa_64"
+    agency = "Chicago Special Service Area #20 Walden Parkway"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.mpbhba.org/business-resources/"]
+
+    def parse(self, response):
+        meetings = self._parse_block(response)
+        for item in meetings:
+            meeting = Meeting(
+                title='Morgan Park Commission',
+                description='',
+                classification=COMMISSION,
+                start=self._parse_start(item),
+                end=None,
+                all_day=False,
+                time_notes="",
+                location=self._parse_location(item),
+                links=[],
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_block(self, response):
+        """Parse data in specific SSA 20 meetings block."""
+        result = []
+        aux = []
+        meetings = []
+        count = 0
+        general_block = response.xpath(
+            '//*[@class="et_pb_module et_pb_text et_pb_text_2 '
+            'et_pb_text_align_left et_pb_bg_layout_light"]'
+        )
+        years = general_block.xpath('.//*[@class="et_pb_text_inner"]/h3/text()').extract()
+        specific_block = general_block.xpath('.//*[@class="et_pb_text_inner"]'
+                                             ).extract_first().split("<strong>")
+        for line in specific_block:
```

Comment:
This was my mistake in the initial issue, but we only want items for SSA 64 here